src/seed.ts

```diff
@@ -6,24 +6,25 @@ export const seed = async (payload: Payload): Promise<void> => {
   const theme = await payload.create<Themes>({ 
     collection: "themes",
     data: {
-      name: "cafe-hugo",
+      name: "donuts",
```

Comment:
Leg doch sonst einfach ein zweites restaurant an in den Seed Datein. Dan hat man als Entwickler auch ein paar mehr optionen zum Testen